rmgpy/tools/fluxtest.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+import os
+import os.path
+import shutil
+
+from rmgpy.tools.fluxdiagram import *
+
+class FluxDiagramTest(unittest.TestCase):
+
+    def test_avi(self):
+        folder = os.path.join(os.getcwd(),'rmgpy/tools/data/flux/')
```

Comment:
could you perhaps use 

```python
folder = os.path.join(os.path.dirname(rmgpy.__file__),'tools/data/flux')
```

rather than `os.getcwd()`

This way, you can run the test from other locations besides the RMG-Py root directory.